### DISCUSSION TITLE
Is the Chart Cell code open sourced?

### DISCUSSION THREAD

**Discussion by maneetgoyal** [observablehq/plot#1529] [Q&A] [answered]
Ref: https://observablehq.com/@observablehq/chart-cell

Was hoping to embed it in a dashboard or just go thru it to figure out how to recreate it in a separate environment outside Observable.

**Comment by mbostock** [comment, marked as answer]
No, it is not. Plot.auto is open-source, but the chart cell UI is part of the Observable platform.
